File: user-service/src/main/java/com/example/user/dto/KakaoLoginResponse.java
package com.example.user.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
@AllArgsConstructor
public class KakaoLoginResponse {

    private String accessToken;
    private String refreshToken;
    private Long userId;
    private String nickname;
    private boolean isNewMember;
}

File: android-client/app/src/main/java/com/foodicircle/data/model/KakaoLoginResponse.kt
package com.foodicircle.data.model

data class KakaoLoginResponse(
    val status: Int,
    val data: LoginData?,
    val message: String?
)

data class LoginData(
    val accessToken: String,
    val refreshToken: String,
    val userId: Long,
    val nickname: String,
    val newMember: Boolean
)
